code/ui/blocks/src/components/ArgsTable/ArgControl.tsx

```diff
@@ -63,7 +65,18 @@ export const ArgControl: FC<ArgControlProps> = ({ row, arg, updateArgs }) => {
   const onBlur = useCallback(() => setFocused(false), []);
   const onFocus = useCallback(() => setFocused(true), []);
 
-  if (!control || control.disable) return <NoControl />;
+  if (!control || control.disable)
+    return isHovered ? (
+      <Link
+        href="https://storybook.js.org/docs/react/essentials/controls"
+        target="_blank"
+        withArrow
+      >
+        Setup controls
+      </Link>
+    ) : (
```

Comment:
Can we find a way to make this Link optional? When rendering action props (aka: onChange, onClick, etc.) they have no control, but they're handled by the actions addon. With this change, even though their control should be empty, we render the link on hover.

I have other examples beyond just action props, but ideally I would like to globally disable any "Setup controls" links with a parameter or something similar.

Reply:
For example, the only reason I need `control: { type: null }` on the below argTypes is to prevent the Link from rendering.

<img width="313" alt="image" src="https://github.com/storybookjs/storybook/assets/37516851/ac8b782e-5dcb-4b20-8269-344e1f3f6078">

Reply:
We use storybook as public interactive docs for our component library. 

The default behavior of rendering the link on hover makes it look like our docs are not completed, and we like to be incredibly thorough.

Reply:
Is there a link I can look at? Generally speaking, I think there's a clear distinction between docs and controls. Docs are specifically meant to document your components where controls is more about testing. But I can understand that some users like you can use controls as a way to document your stories. I'll try to think if there's a way to achieve what you need without docs.